src/liquidity-positions/properties.ts

```diff
@@ -0,0 +1,9 @@
+export enum MigrationSource {
+  SUSHISWAP = 'Sushiswap',
+  V2 = 'V2',
+}
```

Comment:
Do we care to capture V3? Maybe a world where people may still prefer V3 liquidity over V4? Thinking maybe keeping this general (e.g `LiquiditySource`) and keeping V3 would be good, and we can introduce V4 once ready.